app.py

```diff
@@ -38,7 +38,48 @@ def mirror(name):
     }
     return create_response(data)
 
-# TODO: Implement the rest of the API here!
+@app.route('/users', methods = ['GET', 'POST'])
+def users():
+    if request.method == 'GET':
+        if request.args.get('team') is None:
+            data = {
+                'users': db.get('users')
+            }
+        else:
+            matched = [i for i in db.get('users') if i['team'] == request.args.get('team')]
+            data = {
+                'users': matched
+            }
+        return create_response(data)
+    elif request.method == 'POST':
+        input = request.get_json()
+        payload = {'name': input.get('name'),
+                   'age': input.get('age'),
+                   'team': input.get('team')}
+        return create_response(db.create('users', payload))
+
+@app.route('/users/<id>', methods = ['GET', 'PUT', 'DELETE'])
+def userById(id):
+    if request.method == 'GET':
+        if db.getById('users', int(id)) is None:
+            return create_response(None, 404, "User cannot be found")
```

Comment:
If you don't have any result to return back, just leave it blank like `return create_response(status=404, message="User cannot be found")`. The `None` you pass in will override the default value of an empty dictionary from `create_response`